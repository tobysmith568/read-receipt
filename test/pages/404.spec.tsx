import { render } from "@testing-library/react";
import { Provider } from "jotai";
import NotFoundPage from "src/pages/404";

describe("Privacy", () => {
  it("should render the privacy policy", () => {
    const { container } = render(
      <Provider>
        <NotFoundPage />
      </Provider>
    );

    expect(container).toMatchSnapshot();
  });
});
