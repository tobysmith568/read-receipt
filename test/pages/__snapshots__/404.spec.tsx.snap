// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy should render the privacy policy 1`] = `
<div>
  <div
    class="css-1wcee00"
  >
    <a
      class="css-7kai5d"
      href="/"
    >
      &lt;- Home
    </a>
    <div
      class="css-d1odxa"
    >
      <div
        class="css-gv001y"
      >
        <h1>
          404
        </h1>
        <h4>
          Not Found!
        </h4>
      </div>
    </div>
    <div
      class="css-1g71zvr"
    >
      Copyright Â© Toby Smith 
      2022
      .
      <a
        class="css-gy0gih"
        href="/terms"
      >
        Terms.
      </a>
      <a
        class="css-gy0gih"
        href="/privacy"
      >
        Privacy.
      </a>
      <a
        class="css-gy0gih"
        href="/licenses"
      >
        Third-party licenses.
      </a>
    </div>
  </div>
</div>
`;
